.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: Build
+
+on:
+  push:
+    branches: [ main ]
+  release:
+    types: [published, created, edited]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Image Build
+      run: make image-build
+
+    - name: Image Push
+      run: make image-push
```

Comment:
I don't think these will work as written, right? We would need to login to Quay.io for this to work. Does it make more sense to use the push-to-registry? action? https://github.com/marketplace/actions/push-to-registry

Reply:
makes sense.

one thing I'm missing is
username: ${{ secrets.OPDEV_REGISTRY_USER }} 
password: ${{ secrets.OPDEV_REGISTRY_PASSWORD }}

can you or @bcrochet  add these credentials ?   looks like I don't have permission.

Reply:
changes made

Reply:
Doesn't look like I can add secrets to this repository, so it might have to be @jomkz or @tonytcampbell.

Reply:
I have added a new robot account to the Quay [repo](https://quay.io/organization/opdev?tab=robots). Can you see it now @itroyno?